src/ProxyClient/Symfony.php

```diff
@@ -79,4 +84,34 @@ public function refresh($url, array $headers = [])
 
         return $this;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function invalidateTags(array $tags)
+    {
+        $invalidationRequest = $this->options['tags_invalidator']->invalidateTags($tags);
```

Comment:
The tags invalidator optionally returns an `InvalidationRequest`. Invalidation can be performed directly upon the (tag) `ManagerInterface` instance, or on a dedicated `InvalidatorInterface` instance - i.e. the `RemoteInvalidator`.